addons/cargo/XEH_postInit.sqf

```diff
@@ -0,0 +1,5 @@
+#include "script_component.hpp"
+
+["LoadItem", {_this call FUNC(loadItem)}] call EFUNC(common,addEventHandler);
+["UnloadItem", {_this call FUNC(unloadItem)}] call EFUNC(common,addEventHandler);
+["AddCargoItem", {_this call FUNC(addCargoItem)}] call EFUNC(common,addEventHandler);
```

Comment:
Hmm, maybe better would be? 
- `LoadObject`
- `UnloadObject`
- `AddCargoItem`/`AddCargoItemByType`

Up to you, I am bad with names.